<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:rgb(255, 255, 255);
font-size:15px;
font-family:Century Gothic,sans-serif;
}
QFrame{
border:sold 10px rgba(0,0,0);
background-image:url(:/2222.png);

}
QLineEdit{
color:#8d98a1;
background-color:#405361;
font-size:16px;
border-style:outset;
border-radius:10px;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
}
QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
}
QRadioButton{
background:rgba(85,170,255,0);
color:white;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:italic bold;
font-size:14px;

}</string>
    </property>
    <widget class="QPushButton" name="pushButton_Register">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_User">
     <property name="geometry">
      <rect>
       <x>303</x>
       <y>84</y>
       <width>140</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>79</y>
       <width>82</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#0f44f1;&quot;&gt;用户名:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>146</y>
       <width>57</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#0f44f1;&quot;&gt;密码:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Login">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>260</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Password">
     <property name="geometry">
      <rect>
       <x>303</x>
       <y>151</y>
       <width>140</width>
       <height>23</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Teacher">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>教师</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Student">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>学生</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Admin">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>210</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>管理员</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Close">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>0</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/333.png);
background-size:100% 100%;
background-repeat:no-repeat;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_User</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>radioButton_Student</tabstop>
  <tabstop>radioButton_Teacher</tabstop>
  <tabstop>radioButton_Admin</tabstop>
  <tabstop>pushButton_Login</tabstop>
  <tabstop>pushButton_Register</tabstop>
  <tabstop>pushButton_Close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
